mattermost-plugin/webapp/loaders/globalScssClassLoader.js

```diff
@@ -4,7 +4,7 @@
 function blockList(line) {
     return line.startsWith('.focalboard-body') ||
         line.startsWith('.GlobalHeaderComponent') ||
-        line.startsWith('.channel-header__icon .LogoIcon') ||
+        line.startsWith('.boards-rhs-icon') ||
```

Comment:
This wasted a bunch of my time trying to figure out what was going on with the CSS mysteriously vanishing. 
I think this would be much better handled by styled components or at least a liner.

Reply:
Yes, this is to make everything scoped inside focalboard when we load that inside mattermost. It was error prone before because anything that you add could affect mattermost, this way, we only can break our side.